docs/source/means.rst

```diff
@@ -13,7 +13,9 @@ The package provides functions to compute means of different kinds.
 
 .. function:: genmean(x, p)
 
-  Compute the generalized/power mean with exponent ``p`` of ``x``.
+  Compute the generalized/power mean of ``x`` with exponent ``p``,
+  i.e. :math:`\left( \frac{1}{n} \sum_{i=1}^n x_i^p \right)^{\frac{1}{p}}`, where ``n == length(a)``.
```

Comment:
Note you can use that LaTeX syntax even in Julia docstrings (see 6 at http://docs.julialang.org/en/stable/manual/documentation/) when using double backticks, so that there's no difference between the two versions. Also `n = length(x)` would be more logical.